app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -193,9 +190,10 @@ class ThreadFragment : Fragment() {
             toolbarSubject.setTextColor(textColor)
         }
 
-        changeToolbarColorOnScroll(toolbar, messagesListNestedScrollView) { color ->
-            appBar.backgroundTintList = ColorStateList.valueOf(color)
-        }
+        // TODO: Disabled while there is no Tablet design
+        // changeToolbarColorOnScroll(toolbar, messagesListNestedScrollView) { color ->
+        //     appBar.backgroundTintList = ColorStateList.valueOf(color)
+        // }
```

Comment:
We shouldn't remove this, is there no workaround?

Reply:
I didn't find any workaround while there is no Tablet design.

Reply:
Done.